test/ng/snifferSpec.js

```diff
@@ -25,6 +25,25 @@ describe('$sniffer', function() {
     });
 
 
+    it('should be true if running inside nw.js', function() {
```

Comment:
It was just a suggestion - what makes more sense to me. But I don't feel too strongly about it, so feel free to do it the way it makes more sense to you.

##
> As the intend of this PR was to allow history in NW apps

I see it more like restoring the previous behavior of not blocking history on NW apps, which was incorrectly changed while trying to disable it for Chrome PAckaged Apps.

But like I said, totally up to you. As long as the test is there (and is testing the correct thing), it doesn't have t be incorporated into the other one.